dataflow/src/main/java/org/checkerframework/dataflow/cfg/builder/CFGTranslationPhaseThree.java

```diff
@@ -164,6 +167,10 @@ protected static void mergeConsecutiveBlocks(ControlFlowGraph cfg) {
           Block succ = b.getRegularSuccessor();
           if (succ.getType() == BlockType.REGULAR_BLOCK) {
             RegularBlockImpl rs = (RegularBlockImpl) succ;
+            if (rs.getRegularSuccessor() == rs) {
+              // An infinite loop, do not try to merge.
```

Comment:
I think this is an infinite loop at run time, but just a loop in the CFG.  Is that correct?

Reply:
I _think_ a `RegularBlock` can only have one successor in the CFG.  So this is both a loop in the CFG and an infinite loop at run time.  Should I update the comment to that effect?

Reply:
You are right about `RegularBlock` successors.  A small tweak to the comment would be good -- "infinite loop" doesn't really apply to a CFG (or other) graph, so the wording took me aback.

Thanks again.